packages/perseus/src/widgets/interactive-graphs/graphs/polygon.tsx

```diff
@@ -96,9 +99,8 @@ const PolygonGraph = (props: Props) => {
 
     // Logic to build the dragging experience. Primarily used by Limited Polygon.
     const dragReferencePoint = points[0];
-    const constrain = ["angles", "sides"].includes(snapTo)
-        ? (p) => p
-        : (p) => snap(snapStep, p);
+    const constrain: KeyboardMovementConstraint =
```

Comment:
Nit: Can this be a function named `getConstraintFn` that includes a switch statement for the snapTo type? I think that might be easier to follow for engineers looking at this in the future.

Something like 
```
function getConstraintFn(): KeyboardMovementConstraint {
  switch(snapTo) {
    case "angles":
      return p => p
    case "sides":
      return getSideSnapConstraint(...)
    default: // "grid"
      return p => snap(snapStep, p);
  }
}
``

And then you can pass just this function into `constrain` below.